### DISCUSSION TITLE
Date Time picker

### DISCUSSION THREAD

**Discussion by obrien-avatech** [ditdot-dev/vue-flow-form#257] [Ideas] [answered]
Is there a way to incorporate a date time picker into a question? im trying to add one to the date question but having no such luck

**Comment by obrien-avatech** [comment]
Please remove, no longer using

**Comment by EkaterinaVu** [comment, marked as answer]
Hi @obrien-avatech,

it's a good idea and we'll consider implementing this feature in the future.
